internal/trash/trash.go

```
@@ -0,0 +1,378 @@
+			ts, ok := parseStamp(sd.Name())
+			if !ok {
+				continue
+			}
+			stampDir := filepath.Join(base, sd.Name())
+			err := filepath.WalkDir(stampDir, func(p string, d os.DirEntry, werr error) error {
+				if werr != nil || d.IsDir() {
+					return nil //nolint:nilerr // a partial listing beats none
+				}
+				rel, rerr := filepath.Rel(stampDir, p)
+				if rerr != nil {
+					return nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 282) but it returns nil

(nilerr)